src/torchcodec/decoders/_audio_decoder.py

```diff
@@ -37,3 +38,70 @@ def __init__(
         ) = get_and_validate_stream_metadata(
             decoder=self._decoder, stream_index=stream_index, media_type="audio"
         )
+        assert isinstance(self.metadata, core.AudioStreamMetadata)  # mypy
+
+    def get_samples_played_in_range(
+        self, start_seconds: float, stop_seconds: Optional[float] = None
+    ) -> AudioSamples:
```

Comment:
I feel like start_seconds default to the stream's beginning, but that can be done later.

Reply:
If we do that, we should also mirror that in the video API. We _could_ try to play the same trick that `range()` does, where the semantics of the first parameter changes based on the number of parameters, but I'd (softlly) rather not do that.

Reply:
Agreed, I'll update https://github.com/pytorch/torchcodec/issues/150 eventually with more of these defaults. That would be a good onboarding PR.